[2022-09-06T15:01:05.309+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2022-08-30T00:00:00+00:00 [queued]>
[2022-09-06T15:01:05.360+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2022-08-30T00:00:00+00:00 [queued]>
[2022-09-06T15:01:05.360+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-06T15:01:05.361+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 6
[2022-09-06T15:01:05.362+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-06T15:01:05.423+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): postgres_to_s3> on 2022-08-30 00:00:00+00:00
[2022-09-06T15:01:05.439+0000] {standard_task_runner.py:52} INFO - Started process 747 to run task
[2022-09-06T15:01:05.463+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks_v01', 'postgres_to_s3', 'scheduled__2022-08-30T00:00:00+00:00', '--job-id', '568', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_hooks.py', '--cfg-path', '/tmp/tmpdatxxrd2', '--error-file', '/tmp/tmp1mgpk99k']
[2022-09-06T15:01:05.464+0000] {standard_task_runner.py:80} INFO - Job 568: Subtask postgres_to_s3
[2022-09-06T15:01:05.640+0000] {task_command.py:371} INFO - Running <TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2022-08-30T00:00:00+00:00 [running]> on host 45495cb94d52
[2022-09-06T15:01:05.921+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=James
AIRFLOW_CTX_DAG_ID=dag_with_postgres_hooks_v01
AIRFLOW_CTX_TASK_ID=postgres_to_s3
AIRFLOW_CTX_EXECUTION_DATE=2022-08-30T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-30T00:00:00+00:00
[2022-09-06T15:01:05.967+0000] {base.py:68} INFO - Using connection ID 'postgres_localhost' for task execution.
[2022-09-06T15:01:05.983+0000] {dag_with_postgres_hooks.py:29} INFO - save orders data in text file get_orders.txt
[2022-09-06T15:01:05.983+0000] {python.py:173} INFO - Done. Returned value was: None
[2022-09-06T15:01:06.030+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks_v01, task_id=postgres_to_s3, execution_date=20220830T000000, start_date=20220906T150105, end_date=20220906T150106
[2022-09-06T15:01:06.099+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-06T15:01:06.167+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
